packages/mattermost-redux/src/selectors/entities/channels.ts

```diff
@@ -336,6 +336,20 @@ export const getChannelSetInCurrentTeam: (state: GlobalState) => string[] = crea
     },
 );
 
+export const getChannelSetAllTeam: (state: GlobalState) => string[] = createSelector(
```

Comment:
not entirely sure, but the naming here seems a bit off. Wouldn't this be better as `getChannelSetForWholeTeam`?

Reply:
``getChannelSetForWholeTeam`` doesn't sound appropriate to me may be ``getChannelSetForAllTeam as whole is still giving the sense of single team.